doc/manual/src/contributing/testing.md

```diff
@@ -10,6 +10,13 @@ under `src/{library_name}/tests` using the
 You can run the whole testsuite with `make check`, or the tests for a specific component with `make libfoo-tests_RUN`.
 Finer-grained filtering is also possible using the [--gtest_filter](https://google.github.io/googletest/advanced.html#running-a-subset-of-the-tests) command-line option, or the `GTEST_FILTER` environment variable.
 
+### Characterization testing
```

Comment:
I gather this is the same as “Golden tests”, with the latter being (as far as I know) much more common. Maybe rename it?

Reply:
I went with that name before seen it was the one Wikipedia went with. See what I have since linked "below" too.